tests/conftest.py

```diff
@@ -1,11 +1,64 @@
+import os
+import json
+from pathlib import Path
+from typing import Dict
+
 import pytest
-import connexion
 
-from anyvar.app import create_app
+from anyvar.restapi.webapp import create_app
+
+
+def pytest_collection_modifyitems(items):
+    """Modify test items in place to ensure test modules run in a given order.
+    When creating new test modules, be sure to add them here.
+    """
+    MODULE_ORDER = [
```

Comment:
this can be done in a separate issue if you want, but it might be good to ensure that `len(MODULE_ORDER)` == # of files in tests directory that start with tests_